contracts/base/InterestConsensus.sol

```diff
@@ -0,0 +1,150 @@
+/*
+    Copyright 2020 Fabrx Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+pragma solidity 0.5.17;
+pragma experimental ABIEncoderV2;
+
+// Libraries
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+// Interfaces
+import "../interfaces/LendersInterface.sol";
+
+// Contracts
+import "./Initializable.sol";
+import "./Consensus.sol";
+
+
+contract InterestConsensus is Initializable, Consensus {
+    using SafeMath for uint256;
+
+    LendersInterface public lenderInfo;
+
+    // mapping of (lender, blockNumber) to the aggregated node submissions for their request
+    mapping(address => mapping(uint256 => ZeroCollateralCommon.AggregatedInterest)) nodeSubmissions;
+
+    event InterestSubmitted(
```

Comment:
Please, consider extracting events/funcntions to an Interface (maybe InterestConsensusInterface / ConsensusInterface).

Reply:
I think it will have to be `InterestConsensusInterface` will do now